src/components/InstructorRevenueCard/components/LineChart/index.js

```diff
@@ -4,41 +4,40 @@ import hexToRgba from 'hex-rgba'
 import {Line} from 'react-chartjs-2'
 import numberFormattingByType from 'utils/numberFormattingByType'
 
-const sharedOptions = {
+const tooltipColor = '#63768d' // # dark-gray
```

Comment:
Will be able to replace hard-coded color here once https://github.com/eggheadio/tachyons-egghead/issues/28 is done